### PR TITLE
Make font types in search and filters modal consistent

### LINKED ISSUE #1815: Make font types in search and filters modal consistent
### Overview

We need to make fonts on the search and filters modal consistent so users don't get confused about how the modal works

### Action Items

- [x] Grab the color, font size, and font weight for "Request Types" in the Search and Filters Modal
  - loc: `/components/main/Desktop/TypeSelector/index.js` , line: 22, 83

- [x] Apply that color, font size, and font weight to "Boundaries"
  - loc: `/components/main/Desktop/CouncilSelector/index.js` 
  - [x] 1. Add class style from collapsible section - __ADD: Header class style for "Boudaries"__
  - [x] 2. Update class `classes.label` to `classes.header` for "Boundaries" title
  - [x] 3. Cleanup, Delete class style from collapsible section - __DELETE: Label class style for "Boundaries"__

- [x] Apply that color, font size, and font weight to "Date Range"
  - loc: `componenets/DataSelector/DataSelector.jsx`
  - [x] 1.  Add useStyle & class style from collapsible section - __ADD: useStyle const for "Date Range"__
  - [x] 2. Delete `import useStyle from ...`
  - [x] 3. Add `import makeStyles from '@mui/styles/makeStyles';`
  - [x] 4. Update `span` to `div` for the title value ` Date Range&nbsp;` 
  - [x] 5. Update `className={label}` to `className={classes.header}`

### Collapsible Sections

<details>
  <summary>ADD: Header class style for "Boudaries"</summary>
  
```
  header: {
    fontSize: '12.47px',
    fontWeight: theme.typography.fontWeightMedium,
    marginBottom: '8px',
  },
```

</details>

<details>
  <summary>DELETE: Label class style for "Boundaries"</summary>
  
```
  label: {
    display: 'inline-block',
    font: theme.typography.b2,
    marginBottom: '10px',
    color: theme.palette.secondary.light,
  },
```

</details>

<details>
  <summary>ADD: useStyle const for "Date Range"</summary>
  
```
const useStyles = makeStyles(theme => ({
  header: {
    fontSize: '12.47px',
    fontWeight: theme.typography.fontWeightMedium,
    marginBottom: '8px',
  },
}));
```

</details>

### Resources/Instructions

- Figma design file [link](https://www.figma.com/files/team/1039983981189012111/311-data?fuid=604439038560392673)

<details><summary>Screenshot before proposed changes</summary>
<p>

image

</p>
</details>

<details><summary>Screenshot after proposed changes</summary>
<p>
image
</p>
</details>

### PR DESCRIPTION
**Pull request by Brandoncyu**
Fixes #1815 

  - [x] Up to date with `main` branch
  - [x] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [ ] All PR Status checks are successful
  - [ ] Peer reviewed and approved

Any questions? See the [getting started guide](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md)

### CODE REVIEW

File: components/DateSelector/DateSelector.jsx [hunk 1]
```diff
@@ -21,6 +21,12 @@ function DateSelector({
   updateEndDate,
 }) {
   const [expanded, setExpanded] = useState(false);
+  const useStyles = makeStyles(theme => ({
+    header: {
+      fontSize: '12.47px',
+      fontWeight: theme.typography.fontWeightMedium,
```
Comment (traycn) [inline components/DateSelector/DateSelector.jsx:27]: Please add `marginBottom: '8px'` to the header obj

File: components/main/Desktop/CouncilSelector/index.js [hunk 2]
```diff
@@ -15,11 +15,9 @@ import SelectedCouncils from './SelectedCouncils';
...
 
 const useStyles = makeStyles(theme => ({
-  label: {
-    display: 'inline-block',
-    font: theme.typography.b2,
-    marginBottom: '10px',
-    color: theme.palette.secondary.light,
+  header: {
+    fontSize: '12.47px',
+    fontWeight: theme.typography.fontWeightMedium,
```
Comment (traycn) [inline components/main/Desktop/CouncilSelector/index.js:20]: Please add `marginBottom: '8px'` to the header obj

**Review by traycn — COMMENTED** [review]
Left comments on adding marginBottom to the 2 modified title.

File: components/DateSelector/DateSelector.jsx [hunk 3]
```diff
@@ -25,6 +25,7 @@ function DateSelector({
     header: {
       fontSize: '12.47px',
       fontWeight: theme.typography.fontWeightMedium,
+      marginBottom: '8px',
```
Comment (traycn) [inline components/DateSelector/DateSelector.jsx:28]: 1 small tidying task:
- [ ] Remove the unused `label` prop in this file, line 50

1 aligning fix:
We can align the Tooltip that's next to the "Date Range" title by the following:

- [ ] Copy/paste into this files `const useStyle` the `iconStyle` CSS class from the "Request Type" located at the following.
loc: `/components/main/Desktop/TypeSelector/index.js` , line: 19

  - The `iconStyle={iconStyle}` is already being referenced in `ArrowToolTip` DOM element so no additional changes are necessary. Feel free to test on your local to check that the tooltip is vertically aligned with the title.

**Review by traycn — COMMENTED** [review]
Pulled most recent commit and the margin spacings looking great.

Leaving 1 tidying task, and 1 suggestion for aligning the tooltip.

File: components/DateSelector/DateSelector.jsx (hunk 3 above)
Comment (Brandoncyu) [reply components/DateSelector/DateSelector.jsx:31]: done

**Review by traycn — APPROVED** [review]
LGTM!

**Review by ryanfchase — CHANGES_REQUESTED** [review]
I was having a look at the site based on the changes. There's only one aspect that I think still needs adjustment. I think the "Request Types" label is `Roboto, sans-serif`, and the others are "Helvetica Neue, sans-serif".

I can provide more detailed instruction in a moment, but I am noticing a difference in the inspector.

Request Types inspected:
image


Boundaries, inspected:
image


Date Range, inspected:
image

**Comment by ryanfchase** [comment]
Alright, I've just had a look at the figma. Looks like all of those filter section labels should be `Roboto, sans-serif`.

Also I've noticed that the font size should be 14px. We could fix this now, or save it for a following ticket that addresses more font-size discrepancies.

<details><summary>Figma Details for Filters Section Label</summary>
<p>

> Note: it says Neighborhood Districts, but this has since been changed to boundaries. This is an ongoing issue with the design system, we're working on addressing this...

Observe that the font details say: Roboto, Medium. 14px.

image


</p>
</details>

**Comment by Brandoncyu** [comment]
> Alright, I've just had a look at the figma. Looks like all of those filter section labels should be `Roboto, sans-serif`.
> 
> Also I've noticed that the font size should be 14px. We could fix this now, or save it for a following ticket that addresses more font-size discrepancies.
> 
> Figma Details for Filters Section Label

Turns out the secret to this is the "typography" class. It is after I add it that all the classes look consistent.

**Review by ryanfchase — APPROVED** [review]
Good changes! Approved, merge when ready 👍

**Pull request merged**